benchmark/databricks/run_benchmark.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+cluster_type=$1
+
+if [[ $cluster_type == "gpu" ]]; then
+    num_cpus=0
+    num_gpus=2
+elif [[ $cluster_type == "cpu" ]]; then
+    num_cpus=16
+    num_gpus=0
+else
+    echo "unknown cluster type $cluster_type"
+    echo "usage: ./$script_name cpu|gpu"
+    exit 1
+fi
+
+source benchmark_utils.sh
+
+#BENCHMARK_DATA_HOME=/spark-rapids-ml/benchmarking/datasets
+BENCHMARK_DATA_HOME=s3a://spark-rapids-ml-bm-datasets-public
+
+# creates cluster and sets CLUSTER_ID equal to created cluster's id
+create_cluster $cluster_type
+
+# cpu options. reset below if gpu cluster is used
+# kmeans and random forest take a long time on cpu cluster, so do 1 run each
+# do 3 runs in all other cases
+
+n_streams=""
+kmeans_runs=1
+rf_runs=1
+rf_cpu_options="--subsamplingRate=0.5"
+
+if [[ $cluster_type == "gpu" ]]
+then
+    n_streams="--n_streams 4"
+    kmeans_runs=3
+    rf_runs=3
+    rf_cpu_options=""
+fi
+num_runs=3
+
+sep="=================="
+
+echo
+echo "$sep algo: kmeans $sep"
+for i in `seq $kmeans_runs`; do run_bm kmeans \
+--num_runs 1 \
```

Comment:
Could just set `--num-runs $kmeans_runs` in theory (here and elsewhere in fine), unless you need to iterate in bash script explicitly.

Reply:
Things become unstable for some reason with some of the algos, with task failures and longer run times.   We'll have to root cause that.   Could have added the inter-iteration-sleep in the python scripts but went this route.